src/macaron/output_reporter/templates/macaron.html

```diff
@@ -164,6 +164,30 @@
 {% endcall %}
 {% endmacro %}
 
+{% macro render_run_checks(check_tree, excluded_checks) %}
+<ul class="tree-view-nested-list">
+    {% for key, first_level in check_tree.items() %}
+    <li>
+        <div>{{ key }}</div>
+        <ul>
+            {% for key, next_level in first_level.items() recursive %}
```

Comment:
This block handle the second level of checks onward (i.e. the checks that have parent).